crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1338,8 +1338,8 @@ fn visit_node(
             path: fill_paths,
             stroke: stroke_paths,
             stroke_cap: node.stroke_cap.clone(),
-            start_angle_degrees: arc_data.starting_angle,
-            sweep_angle_degrees: arc_data.ending_angle,
+            start_angle_degrees: euclid::Angle::radians(arc_data.starting_angle).to_degrees(),
+            sweep_angle_degrees: euclid::Angle::radians(arc_data.ending_angle).to_degrees(),
```

Comment:
What does this fix?

Reply:
I think these values are in radians from the plugin. I noticed this when I tried animating arcs; dials/gauges still works (so either I made a similar change there, or it wasn't needed).